py/server/deephaven/table.py

```diff
@@ -694,6 +696,63 @@ def to_string(self, num_rows: int = 10, cols: Union[str, Sequence[str]] = None)
         except Exception as e:
             raise DHError(e, "table to_string failed") from e
 
+    def diff(self, table: Table, max_lines: int = 1, floating_inexact: bool = False,
+             floating_fraction: bool = False, ignore_column_order: bool = False) -> str:
+        """Returns the differences between this table and the provided table as a string.
+
+        This method starts by comparing the table sizes, and then the schema of the two tables, such as the number of
+        columns, column names, column types, column orders. If the schemas are different, the comparison stops and
+        the differences are returned. If the schemas are the same, the method proceeds to compare the data in the
+        tables. The method compares the data in the tables column by column and only records the first difference
+        found in each column.
```

Comment:
What does `only records the first difference found in each column` mean?

Reply:
It means that if a column in the two tables differ on more than 1 cell, only the first different cell is noted and the comparison for the given column is concluded.